content/webinars/aws-lambda-api-gateway/index.md

```diff
@@ -0,0 +1,62 @@
+---
+# Name of the webinar.
+title: "API Gateway for an AWS Lambda"
+meta_desc: "Search Description"
```

Comment:
Is this the right `meta_desc` for this?

Reply:
Nope, thanks for catching this. Updated the description.